packages/services/src/assets/sip10-tokens.utils.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  CryptoAssetCategories,
+  CryptoAssetChains,
+  CryptoAssetProtocols,
+  Sip10CryptoAssetInfo,
+} from '@leather.io/models';
+import { getTicker, isUndefined } from '@leather.io/utils';
+
+import { HiroFtMetadataResponse } from '../infrastructure/api/hiro/hiro-stacks-api.client';
+
+export function isTransferableSip10Token(token: Partial<HiroFtMetadataResponse>) {
+  return !isUndefined(token.decimals) && !isUndefined(token.name) && !isUndefined(token.symbol);
+}
+
+export function getAssetNameFromIdentifier(assetIdentifier: string) {
+  return !assetIdentifier.includes('::') ? assetIdentifier : assetIdentifier.split('::')[1];
+}
+
+export function getContractPrincipalFromAssetIdentifier(assetIdentifier: string) {
```

Comment:
`getContractPrincipalFromAssetIdentifier` and `getAddressFromAssetIdentifier` seem to have the same logic?